.github/workflows/sync_labeler.yml

```
@@ -0,0 +1,32 @@
+  pull_request_target:
+    types: [opened, edited, synchronize]
+
+jobs:
+  labeler:
+    name: Sync PR Labels
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - name: "File Based Labeler"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)